parsing_scripts/lamoda_parser.py

```diff
@@ -26,58 +26,76 @@ async def parse(self):
         soup = await self.get_soup(self.BASE_URL)
 
         # find all links with product categories
-        a_list = soup.find_all('a', attrs={'class': f'{self.category_class}'})
+        a_list = soup.find_all("a", attrs={"class": f"{self.category_class}"})
 
         # in case of one product can be in several categories we'll store visited product's links
         # to be sure not to visit one product twice
         visited_products = set()
-        for a in a_list[:10]:  # slice is for test version
-            actual_url = self.BASE_URL + a.attrs['href']
+        for a in a_list[:5]:  # slice is for test version
+            actual_url = self.BASE_URL + a.attrs["href"]
             actual_soup = await self.get_soup(actual_url)
 
```

Comment:
try to minimize iterations while parsing, you can use XPATH for advanced element picking, read some articles